File: c/ABC081A/main.c
#include <stdio.h>

int main(void) {
    char a,b,c;
    int result = 0;
    scanf("%c%c%c",&a, &b, &c);
    if (a == '1') {
        result += 1;
    }
    if (b == '1') {
        result += 1;
    }
    if (c == '1') {
        result += 1;
    }
    printf("%d\n", result);
    return 0;
}

File: rust/abc081a/src/main.rs
// -*- coding:utf-8-unix -*-

use proconio::input;

// ABC081A - Placing Marbles
// https://atcoder.jp/contests/abs/tasks/abc081_a

fn main() {
    input! {
        n: String,
    }
    let v:Vec<char> = n.chars().collect();
    let mut result =  if v[0] == '1' { 1 } else {0 };
    result +=  if v[1] == '1' { 1 } else { 0 };
    result +=  if v[2] == '1' { 1 } else { 0 };
    
    println!("{}", result);
}
